setup.py

```diff
@@ -14,10 +14,15 @@ def find_version():
             raise RuntimeError("Unable to find version string.")
 
 
+with open('README.rst') as f:
+    long_description = f.read()
+
+
 setup(
     name='auth0-python',
     version=find_version(),
     description='Auth0 Python SDK',
+    long_description=long_description,
     author='Auth0',
```

Comment:
I'm not familiar with this but I just checked a blogpost and I think it's missing a line like this:

```
    long_description_content_type='text/markdown'
```

Reply:
That's a new thing which allows a Markdown README.md to be used:

* https://dustingram.com/articles/2018/03/16/markdown-descriptions-on-pypi

But RST files are the default.

Here's an example that has a README.rst:

* https://github.com/pypa/pip

They have just a `long_description=long_description,` line in `setup()`:

* https://github.com/pypa/pip/blob/master/setup.py#L37

And their PyPI page shows their README.rst:

* https://pypi.org/project/pip/